proof.go

```diff
@@ -142,8 +147,11 @@ func (proof Proof) VerifyNamespace(h hash.Hash, nID namespace.ID, data [][]byte,
 	return proof.verifyLeafHashes(nth, true, nID, gotLeafHashes, root)
 }
 
-func (proof Proof) verifyLeafHashes(nth *Hasher, verifyCompleteness bool, nID namespace.ID, gotLeafHashes [][]byte, root []byte) bool {
+// verifyLeafHashes checks whether all the leaves matching the namespace ID nID are covered by the proof if verifyCompleteness is set to true
+func (proof Proof) verifyLeafHashes(nth *Hasher, verifyCompleteness bool, nID namespace.ID, leafHashes [][]byte, root []byte) bool {
 	var leafIndex uint64
+	// TODO [Me] Why called leftSubtrees?
```

Comment:
Not sure what would be a better name. LeftSubtreeRoots maybe?

Reply:
My question was originally about why the the data populated inside `leftSubtrees` represent the left subtrees. Later, by examining the code, I figured why. I should have removed this comment. 
Nevertheless, I liked your suggested name i.e., `LeftSubtreeRoots`, it is more indicative of the content of the variable.